select.go

```diff
@@ -238,8 +342,10 @@ func (e *ESql) convertValExpr(expr *sqlparser.Expr) (dsl string, err error) {
 	case *sqlparser.SQLVal:
 		dsl = sqlparser.String(*expr)
 		dsl = strings.Trim(dsl, `'`)
+	case sqlparser.ValTuple:
```

Comment:
This type looks specious, reader may wonder why previous one is a pointer but not this one

Reply:
That comes from the sqlparser definition. Seems ValTuple is one of the several special cases.
I just add a comment to this line.